ship/src/main/scala/ch/epfl/bluebrain/nexus/ship/Main.scala

```diff
@@ -56,13 +56,23 @@ object Main
       }
       .map(_.as(ExitCode.Success))
 
-  private[ship] def run(r: RunCommand) =
-    for {
-      (config, eventsStream) <- InitShip(r)
-      _                      <- Transactors.init(config.database).use { xas =>
-                                  RunShip(eventsStream, config.input, xas)
-                                }
-    } yield ()
+  private[ship] def run(r: RunCommand): IO[Unit] = {
+    val clock = Clock[IO]
+    InitShip(r).use { case (config, eventsStream, xas) =>
+      for {
+        start         <- clock.realTimeInstant
+        reportOrError <- Transactors
+                           .init(config.database)
+                           .use { xas =>
+                             RunShip(eventsStream, config.input, xas)
+                           }
+                           .attempt
```

Comment:
Why not use the existing transactors?

Reply:
Good catch, I missed to remove the previous one